Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3745,6 +3745,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyMarginBlock: // logical shorthand
         case CSSPropertyMarginInline: // logical shorthand
         case CSSPropertyMarker:
+        case CSSPropertyMasonryAutoFlow:
```

Comment:
Please file a bug to add discrete animation support for `masonry-auto-flow`.

Reply:
Done https://bugs.webkit.org/show_bug.cgi?id=248646